app/src/main/kotlin/co/netguru/baby/monitor/client/feature/recording/RecordingController.kt

```diff
@@ -26,10 +29,11 @@ class RecordingController @Inject constructor(
             .map {
                 val data = when {
                     isMachineLearningWithEnoughData(it) -> RecordingData.MachineLearning(
-                        it.first.takeLast(MachineLearning.DATA_SIZE * 2).toByteArray(),
+                        it.first.takeLast(FeedbackController.DATA_SIZE * 2).toByteArray(),
                         it.second.takeLast(MachineLearning.DATA_SIZE).toShortArray()
                     )
                     isNoiseDetectionWithEnoughData(it) -> RecordingData.NoiseDetection(
+                        it.first.takeLast(FeedbackController.DATA_SIZE * 2).toByteArray(),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/7c069c26782df05dac8a097aa956e351ad95f920/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/recording/RecordingController.kt#L36"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>